src/json/jobs/JobParams.ts

```diff
@@ -0,0 +1,198 @@
+import { JSONSchema6 } from "json-schema";
+
+const legacyExporParams: JSONSchema6["properties"] = {
+  createdSince: {
+    type: "integer",
+    title: "Created Since"
+  },
+  createdBefore: {
+    type: "integer",
+    title: "Created Before"
+  },
+  updatedSince: {
+    type: "integer",
+    title: "Updated Since"
+  },
+  updatedBefore: {
+    type: "integer",
+    title: "Updated Before"
+  },
+  createdOrUpdatedSince: {
+    type: "integer",
+    title: "Created Or Updated Since"
+  },
+  createdOrUpdatedBefore: {
+    type: "integer",
+    title: "Created Or Updated Before"
+  }
+};
+
+const rewardBalanceParams: JSONSchema6["properties"] = {
+  createdSince: {
+    type: "integer",
+    title: "Created Since"
+  },
+  createdBefore: {
+    type: "integer",
+    title: "Created Before"
+  }
+};
+
+const redeemableRewardBalanceParams: JSONSchema6 = {
+  properties: {
+    filter: {
+      type: "object",
+      title: "Redeemable Reward Balance Filter",
+      properties: {
+        type_eq: {
+          type: "string",
+          enum: ["PCT_DISCOUNT", "CREDIT", "FUELTANK", "INTEGRATION"]
+        },
+        unit_eq: {
+          type: "string"
+        },
+        unit_in: {
+          type: "array",
+          items: {
+            type: "string"
+          }
+        },
+        unitType_eq: {
+          type: "string"
+        },
+        currency_eq: {
+          type: "string"
+        }
+      }
+    }
+  }
+};
+
+const mutationEvaluationOptions: JSONSchema6 = {
+  $schema: "http://json-schema.org/draft-06/schema#",
+  type: "object",
+  properties: {
+    webhooks: {
+      oneOf: [
+        {
+          type: "boolean",
+          title: "Webhooks Enabled",
+          description: "If true then all triggered webhook types will be sent"
+        },
+        {
+          type: "object",
+          properties: {
+            enabledWebhookTypes: {
+              type: "array",
+              title: "Enabled Webhook Types",
+              items: {
+                type: "string",
+                enum: [
+                  "user.created",
```

Comment:
@jericmason This set of webhook types probably could be centralized. I'm concerned about maintainability, and remembering to update this when we add a new webhook type.

Reply:
Makes sense, I could move them to a central location; but they may require some processing before including them here since the list won't include the export started/completed webhooks.